.github/workflows/release.yml

```diff
@@ -0,0 +1,130 @@
+name: 'CD'
+
+on:
+  workflow_call:
+    inputs:
+      flavor:
```

Comment:
Q: What is a flavor?

Sorry, this is probably just my lack of knowledge. But maybe it's worth adding some examples or something elaboration for future maintainers.